src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -1973,71 +1974,118 @@ void Player_ZeroRootLimbYaw(Player* this) {
     this->skelAnime.jointTable[1].y = 0;
 }
 
-void func_80832DBC(Player* this) {
+/**
+ * Finishes "AnimMovement" by resetting various aspects of Player's SkelAnime structure.
+ *
+ * This function is called in `Player_SetupAction` so it will run on every action change.
+ * Therefore it is not required to be called manually in most cases. Simply changing actions
+ * will implicitly end AnimMovement.
+ * There are some specific cases where it can be desirable to call this function mid-action.
```

Comment:
what are these "specific cases"? Maybe something like

> This function is in `Player_SetupAction` so it will run on every action change, but it could also be called in action funcs to change animations in the middle of an action.